next-level/lib/checkers/rust.py

```
@@ -0,0 +1,97 @@
+                timeout=15,
+            )
+            result["formatted"] = proc.returncode == 0
+        except (subprocess.TimeoutExpired, FileNotFoundError):
+            pass
+
+    # Strip unnecessary comments
+    try:
+        from comment_stripper import strip_comments
+        strip_result = strip_comments(filepath, "rust")
+        result["comments_stripped"] = strip_result.get("stripped", 0)
+    except (ImportError, Exception):
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)